bin/build_and_run_jar.sh

```diff
@@ -33,7 +33,8 @@ SRC_DIR="portability-$BINARY"
 mvn=$(which mvn)|| { echo "Maven (mvn) not found. Please install it and try again." >&2; exit 1; }
 
 # Copy settings yaml files from ENV/settings/ into $SRC_DIR/src/main/resources/
-SETTINGS_DEST_PATH="$SRC_DIR/src/main/resources/settings/"
+DEST_RESOURCES_PATH="$SRC_DIR/src/main/resources"
```

Comment:
can you name these vars consistently re where _DEST_ occurs in the variable name?

Reply:
Done